### PR TITLE
[14.0][FIX] added case when resupply from another warehouse

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

**Comment by sorvis** [comment]
@ChrisOForgeFlow when I'm running gitagg it is getting a merge conflict on __manifest__.py and I see here github is saying there is a conflict. What is the normal process to resolve this should the conflict be fixed here so I can pull this in directly with gitagg?

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow when I'm running gitagg it is getting a merge conflict on **manifest**.py and I see here github is saying there is a conflict. What is the normal process to resolve this should the conflict be fixed here so I can pull this in directly with gitagg?

I think if this pr is approved, you can update core repos.yml, another pr related with your aggregate should be rebased to continue on this cases
ping @JordiBForgeFlow @LoisRForgeFlow

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow when I'm running gitagg it is getting a merge conflict on **manifest**.py and I see here github is saying there is a conflict. What is the normal process to resolve this should the conflict be fixed here so I can pull this in directly with gitagg?

Rebase was done, please try again

File: ddmrp/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "DDMRP",
     "summary": "Demand Driven Material Requirements Planning",
-    "version": "14.0.1.13.3",
```
Comment (LoisRForgeFlow) [inline ddmrp/__manifest__.py]: Can you rebase and refresh the number update here and in the migration script?

File: ddmrp/migrations/14.0.1.13.2/post-fix_distributed_source_location.py
Comment (LoisRForgeFlow) [inline ddmrp/migrations/14.0.1.13.2/post-fix_distributed_source_location.py:1]: move the script to a new version accordingly after the rebase

File: ddmrp/models/stock_buffer.py [hunk 2]
```diff
@@ -1731,6 +1731,27 @@ def _source_location_from_route(self, route=None):
             )
             if rule.procure_method == "make_to_stock":
                 return rule.location_src_id
+            elif rule.procure_method == "make_to_order":
+                # If resupply from another warehouse, this rule can't be retrieved by
+                # method _get_rule, because that we try to get this rule bases on previous rule
+                pull_rule = self.env["stock.rule"].search(
```
Comment (LoisRForgeFlow) [inline ddmrp/models/stock_buffer.py:1761]: Can you add a test case to illustrate the use case and ease migration/fwports?

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks!

**Comment by LoisRForgeFlow** [comment]
Merging without bumping version as it has been done manually.

/ocabot merge nobump

**Comment by LoisRForgeFlow** [comment]
@ChrisOForgeFlow Could you fwport this fix to v15?

**Pull request merged**